src/activity_tracker_handler.py

```diff
@@ -0,0 +1,162 @@
+from datetime import datetime
+from src import consts
+import pandas as pd
+import logging
+import re
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+# Delete not necessary events from activity-tracker data
+# Necessary events can see in the const file: ACTIVITY_TRACKER_EVENTS and ACTION_EVENTS
+def __filter_ati_data(ati_data: pd.DataFrame):
+    event_types = [consts.ACTIVITY_TRACKER_EVENTS.ACTION.value,
+                   consts.ACTIVITY_TRACKER_EVENTS.COMPILATION_FINISHED.value]
+    ati_data = ati_data[(ati_data[consts.ACTIVITY_TRACKER_COLUMN.EVENT_TYPE.value].isin(event_types))
+                        & (ati_data[consts.ACTIVITY_TRACKER_COLUMN.EVENT_DATA.value].isin(
+        consts.ACTIVITY_TRACKER_EVENTS.action_events()))]
+    return ati_data
+
+
+# Delete : symbol from hours in timestamp for correcting convert to datetime
+# For example 2019-12-09T18:41:28.548+03:00 -> 2019-12-09T18:41:28.548+0300
+def __corrected_time(timestamp: str):
+    return re.sub(r'([-+]\d{2}):(\d{2})(?:(\d{2}))?$', r'\1\2\3', timestamp)
```

Comment:
I want to use the unittest module for testing, but I can not find the best way of testing the private methods (which were marked __). Because when I try to make a class for tests, I can not use the private functions from another module, I can use a global variable in current test module for calling the private functions from another module, but I think it looks not beautiful. Maybe do you know the best way of testing the private functions?
You can see the answer about it, but it doesn't help: https://softwareengineering.stackexchange.com/questions/380287/why-is-unit-testing-private-methods-considered-as-bad-practice

Reply:
I made my variant tests for it. I hope it looks ok

Reply:
Yeah, looks good, thanks!